Userland/Libraries/LibWeb/CSS/ColumnCount.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023, Aliaksandr Kalenik <kalenik.aliaksandr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+namespace Web::CSS {
+
+class ColumnCount {
+public:
+    enum class Type {
+        Auto,
+        Integer
+    };
+
+    ColumnCount(int value)
+        : m_type(Type::Integer)
+        , m_value(value)
+    {
+    }
+    ColumnCount() {};
+
+    static ColumnCount make_auto()
+    {
+        return ColumnCount();
+    }
```

Comment:
Maybe add a `make_integer(int)` function too and hide both constructors, just to make it more explicit?

Otherwise, all good! :+1: